### ISSUE TITLE
DenseNet Imagenet Inference is Wrong

### ISSUE THREAD

**Issue by MSiam** [farizrahman4u/keras-contrib#190]
I have tried using DenseNetImageNet121 on images it keeps getting new predicted label with each time I run on the same image with the same imagenet weights that are used by default.

Although when I am using the same exact code of inference but instead on ResNet50 it works to get the same output label as expected. Is the code by any chance not conditioning dropout on the train phase and is wrongly using it in both train and test?

Thanks
